Sources/KSuite/KSuiteView.swift

```diff
@@ -0,0 +1,89 @@
+/*
+ iOS Features
+ Copyright (C) 2025 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import DesignSystem
+import InfomaniakCoreSwiftUI
+import KSuiteUtils
+import SwiftUI
+
+public struct KSuiteView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    private let configuration: KSuiteConfiguration
+    private let isAdmin: Bool
+
+    public init(configuration: KSuiteConfiguration, isAdmin: Bool) {
+        self.configuration = configuration
+        self.isAdmin = isAdmin
+    }
+
+    public var body: some View {
+        VStack(spacing: IKPadding.huge) {
+            KSuiteResources.background.swiftUIImage
+                .resizable()
+                .scaledToFit()
+                .ignoresSafeArea()
+
+            VStack(alignment: .leading, spacing: IKPadding.huge) {
+                VStack(alignment: .leading, spacing: IKPadding.medium) {
+                    Text(configuration.title)
+                        .multilineTextAlignment(.center)
+                        .font(FontHelper.title)
+                        .foregroundStyle(ColorHelper.primary)
+                        .frame(maxWidth: .infinity, alignment: .center)
+
+                    Text(configuration.description)
+                }
+
+                VStack(alignment: .leading, spacing: IKPadding.medium) {
+                    ForEach(configuration.labels) { label in
+                        Label {
+                            Text(label.text)
```

Comment:
We don't not want to emphasize some part of the text in bold? (kChat / kDrive / 3 To…)